automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceSummaryPage.scala

```diff
@@ -0,0 +1,224 @@
+package org.broadinstitute.dsde.firecloud.page.workspaces
+
+import org.broadinstitute.dsde.firecloud.config.Config
+import org.broadinstitute.dsde.firecloud.page.{PageUtil, _}
+import org.openqa.selenium.WebDriver
+import org.scalatest.selenium.Page
+
+/**
+  * Page class for the Workspace Detail page.
+  */
+class WorkspaceSummaryPage(namespace: String, name: String)(implicit webDriver: WebDriver) extends WorkspacePage with Page with PageUtil[WorkspaceSummaryPage] {
+
+  override val url: String = s"${Config.FireCloud.baseUrl}#workspaces/$namespace/$name"
+
+  override def awaitLoaded(): WorkspaceSummaryPage = {
+    await condition { enabled(testId("submission-status")) || enabled(testId("workspace-details-error")) }
+    this
+  }
+
+  /**
+    * Dictionary of access level labels displayed in the web UI.
+    */
+  object AccessLevel extends Enumeration {
```

Comment:
I feel like I already saw this enum somewhere else -- move to a shared location?